test/testuninitvar.cpp

```diff
@@ -4370,15 +4370,19 @@ class TestUninitVar : public TestFixture {
                             "    arc << p;\n"
                             "    return *p;\n"
                             "}");
-            ASSERT_EQUALS("[test.cpp:4]: (error) Uninitialized variable: p\n", errout.str());
+            ASSERT_EQUALS("[test.cpp:3]: (error) Uninitialized variable: p\n"
+                          "[test.cpp:4]: (error) Uninitialized variable: p\n",
```

Comment:
It seems that ideally we would only warn once.

Reply:
Yea, it would be ideal but this would break a lot of tests in our cfg files which rely on warning multiple times for the same variable.